### ISSUE TITLE
Ability to embed dashboard/widgets

### ISSUE THREAD

**Issue by aliaksandrbasau** [reportportal/reportportal#396]
I'd like to have an ability to generate HTML code for a dashboard or an individual widget so it can be embedded into some other product (e.g. into Confluence page).

**Comment by aliaksandrbasau** [comment]
@DzmitryHumianiuk UI should provide a way to generate HTML code (e.g. iframe) or URL for embedding a dashboard. Here are some examples from other solutions:
- qTest - https://support.qasymphony.com/hc/en-us/articles/115002952606-Embed-Reports
- TestRail - http://docs.gurock.com/testrail-integration/dashboards-confluence
- PowerBI - https://docs.microsoft.com/en-us/power-bi/developer/embedding-content
Usually the bigger issue is that people that are not directly involved into test automation don't go to RP and they don't have visibility into automation.
The good solution is to have all important information in Confluence where you can combine it from different sources. And embedding allows you to display up-to-date information and there is no need to do synchronization manually.

**Comment by varadha2k** [comment]
do we have an update on this issue? I am also looking for similar capability in reportportal.

**Comment by DzmitryHumianiuk** [comment]
@varadha2k we consider to make it.
ETA ~~~Nov-Dec 2018

**Comment by le1ia** [comment]
@DzmitryHumianiuk is this still on track for Dec 2018 timeframe?

**Comment by DzmitryHumianiuk** [comment]
@le1ia `5.0asis` will not have it.
but will be with high priority after it.

**Comment by le1ia** [comment]
@DzmitryHumianiuk Thank you, my team is very much looking forward to this feature :)

**Comment by DzmitryHumianiuk** [comment]
@le1ia we are out of schedule.
completing scope of version 5.0. With minimal scope.

this feature still in backlog

**Comment by mcarriga** [comment]
@DzmitryHumianiuk Just curious, any progress on prioritizing this feature request?

**Comment by DzmitryHumianiuk** [comment]
@mcarriga no estimations for now.
focusing on RPv5 release.

but already had first brainstorm to make it possible

**Comment by evjlobanova** [comment]
Connected with #583

**Comment by yay14** [comment]
> @aliaksandrbasau you need an API endpoint, which will return you HTML code?

Yes, is it possible now? Want to download dashboard as a file using a script and share with customers.

**Comment by DzmitryHumianiuk** [comment]
@yay14 not yeat. But we recently accomplished technical investigation of the options available.
Considered to be a part of 6.x versions.
